boom! testing tokio-postgres-0.1.0 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! c59827ecd7d14e6a36f9eee50a1ecb42701dfb9014145931d9fe9b401a4ab616
boom! running `docker start -a c59827ecd7d14e6a36f9eee50a1ecb42701dfb9014145931d9fe9b401a4ab616`
kablam!    Compiling futures-state-stream v0.1.0
kablam!    Compiling tokio-dns-unofficial v0.1.1
kablam!    Compiling tokio-uds v0.1.1
kablam!    Compiling tokio-postgres v0.1.0 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 19.25 secs
boom! running `docker rm -f c59827ecd7d14e6a36f9eee50a1ecb42701dfb9014145931d9fe9b401a4ab616`
blam! c59827ecd7d14e6a36f9eee50a1ecb42701dfb9014145931d9fe9b401a4ab616
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! a8c26f8fccbfa644ed2de332e9d7f126b001373e1031994c4ef071cc747dfe17
boom! running `docker start -a a8c26f8fccbfa644ed2de332e9d7f126b001373e1031994c4ef071cc747dfe17`
kablam!    Compiling tokio-postgres v0.1.0 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 30.54 secs
boom! running `docker rm -f a8c26f8fccbfa644ed2de332e9d7f126b001373e1031994c4ef071cc747dfe17`
blam! a8c26f8fccbfa644ed2de332e9d7f126b001373e1031994c4ef071cc747dfe17
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! bc8df7c0d79d8a08db295dbf16c6210c82d9b33acbfa295beecdb96e49f2af70
boom! running `docker start -a bc8df7c0d79d8a08db295dbf16c6210c82d9b33acbfa295beecdb96e49f2af70`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/tokio_postgres-48271b062ffe6a71
blam! 
blam! running 17 tests
blam! test test::cancel ... FAILED
blam! test test::batch_execute_err ... FAILED
blam! test test::composite ... FAILED
blam! test test::batch_execute_ok ... FAILED
blam! test test::md5_user ... FAILED
blam! test test::enum_ ... FAILED
blam! test test::domain ... FAILED
blam! test test::md5_user_no_pass ... FAILED
blam! test test::md5_user_wrong_pass ... FAILED
blam! test test::pass_user_wrong_pass ... FAILED
blam! test test::pass_user ... FAILED
blam! test test::pass_user_no_pass ... FAILED
blam! test test::prepare_execute ... FAILED
blam! test test::query ... FAILED
blam! test test::transaction ... FAILED
blam! test test::ssl_user_ssl_required ... FAILED
blam! test test::unix_socket ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test::cancel stdout ----
blam! 	thread 'test::cancel' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::batch_execute_err stdout ----
blam! 	thread 'test::batch_execute_err' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test::composite stdout ----
blam! 	thread 'test::composite' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::batch_execute_ok stdout ----
blam! 	thread 'test::batch_execute_ok' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::md5_user stdout ----
blam! 	thread 'test::md5_user' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::enum_ stdout ----
blam! 	thread 'test::enum_' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::domain stdout ----
kablam! error: test failed
blam! 	thread 'test::domain' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::md5_user_no_pass stdout ----
blam! 	thread 'test::md5_user_no_pass' panicked at 'unexpected error Error communicating with the server: Connection refused (os error 111)', src/test.rs:32
blam! 
blam! ---- test::md5_user_wrong_pass stdout ----
blam! 	thread 'test::md5_user_wrong_pass' panicked at 'unexpected error Error communicating with the server: Connection refused (os error 111)', src/test.rs:46
blam! 
blam! ---- test::pass_user_wrong_pass stdout ----
blam! 	thread 'test::pass_user_wrong_pass' panicked at 'unexpected error Error communicating with the server: Connection refused (os error 111)', src/test.rs:84
blam! 
blam! ---- test::pass_user stdout ----
blam! 	thread 'test::pass_user' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::pass_user_no_pass stdout ----
blam! 	thread 'test::pass_user_no_pass' panicked at 'unexpected error Error communicating with the server: Connection refused (os error 111)', src/test.rs:70
blam! 
blam! ---- test::prepare_execute stdout ----
blam! 	thread 'test::prepare_execute' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::query stdout ----
blam! 	thread 'test::query' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::transaction stdout ----
blam! 	thread 'test::transaction' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! ---- test::ssl_user_ssl_required stdout ----
blam! 	thread 'test::ssl_user_ssl_required' panicked at 'unexpected error Error communicating with the server: Connection refused (os error 111)', src/test.rs:210
blam! 
blam! ---- test::unix_socket stdout ----
blam! 	thread 'test::unix_socket' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/result.rs:870
blam! 
blam! 
blam! failures:
blam!     test::batch_execute_err
blam!     test::batch_execute_ok
blam!     test::cancel
blam!     test::composite
blam!     test::domain
blam!     test::enum_
blam!     test::md5_user
blam!     test::md5_user_no_pass
blam!     test::md5_user_wrong_pass
blam!     test::pass_user
blam!     test::pass_user_no_pass
blam!     test::pass_user_wrong_pass
blam!     test::prepare_execute
blam!     test::query
blam!     test::ssl_user_ssl_required
blam!     test::transaction
blam!     test::unix_socket
blam! 
blam! test result: FAILED. 0 passed; 17 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f bc8df7c0d79d8a08db295dbf16c6210c82d9b33acbfa295beecdb96e49f2af70`
blam! bc8df7c0d79d8a08db295dbf16c6210c82d9b33acbfa295beecdb96e49f2af70
